src/rdfa-helpers.js

```diff
@@ -191,7 +198,8 @@ function rdfaAttributesToTriples(rdfaAttributes) {
       return new Triple({
         subject: triple.object,
         predicate: triple.predicate.slice(1),
-        object: triple.subject
+        object: triple.subject,
+        datatype: 'http://www.w3.org/2000/01/rdf-schema#Resource'
```

Comment:
this fixes a bug where the datatype is not set to resource on inverse triples and doesn't really belong in this PR.